stingray/powerspectrum.py

```diff
@@ -438,7 +438,7 @@ def from_time_array(times, dt, segment_size=None, gti=None, norm="frac",
 
     @staticmethod
     def from_events(events, dt, segment_size=None, norm="frac",
-                    silent=False, use_common_mean=True):
+                    silent=False, use_common_mean=True, gti=None):
```

Comment:
So I'm curious what happens if the EventList or (in the case of the `from_lightcurve` method below) the `Lightcurve` object has GTIs, and `gti` is not `None` here: will this use the GTIs in the `Lightcurve` object? The GTIs passed in here? Do we *need* an option to pass GTIs here? It would be valuable to be clear about this, I think.

Reply:
In the docstring I did write that "These are interesected with the GTIs of the input object". This is basically just an additional level of filtering. We had the option in the old interface, I just kept it